library/core/src/androidTest/java/com/google/android/exoplayer2/upstream/AndroidDataSourceTest.java

```diff
@@ -0,0 +1,31 @@
+package com.google.android.exoplayer2.upstream;
+
+import android.content.Context;
+import android.net.Uri;
+import android.test.InstrumentationTestCase;
+
+public class AndroidDataSourceTest extends InstrumentationTestCase {
```

Comment:
Please split into separate AssetDataSourceTest and ContentDataSourceTest classes. Thanks.